src/ball.rs

```diff
@@ -40,17 +71,33 @@ impl Ball {
     /// translated to (0.0, 0.0).
     ///
     pub fn update(&mut self, input_axes: Vec2, delta_time: f32) {
-        // Update the ball's position
+        let mut rng = rand::thread_rng();
+
+        if self.time_running >= self.velocity_change_time_at {
+            let radians: f32 = rng.gen_range(0.0..2.0 * f32::consts::PI);
```

Comment:
would be nice to abstract this into a function that takes in the length of the new random vector and then does the math and returns it. Could be used in `Self::new` above to prevent having vector constants